.travis.yml

```diff
@@ -5,6 +5,7 @@ node_js:
   - lts/*
   - '10'
 cache:
+  if: $(npm audit --audit-level=moderate) = 1
   npm: false
```

Comment:
[Q] Would we need to print something here to check in logs that cache has been cleared?